include/eld/Core/LinkerScript.h

```diff
@@ -240,6 +240,9 @@ class LinkerScript {
 
   void updateRuleOp(plugin::LinkerWrapper *W, eld::Module *M, RuleContainer *R,
                     ELFSection *S, const std::string &Annotation = "");
+  void updateLinkStatsOp(plugin::LinkerWrapper *W, eld::Module *M,
+                         const std::string &StatName,
+                         const std::string &Annotation = "");
```

Comment:
What is the `Annotation` parameter here? Why does it have a default value? Additionally, what do you think about `StatName` and `StatValue` as the parameter names?